src/App.java

```diff
@@ -1,5 +1,77 @@
+import java.util.Scanner;
+
 public class App {
     public static void main(String[] args) throws Exception {

```

Comment:
I'm never wild about keeping a lot of work in `main()` - I would prefer for your `main()` method to create an `App` object and then call a method on the new object, such as `start()`. It's not a good idea to stay in static methods for too long.